test/test_tree_2d_mhdmultiion.jl

```diff
@@ -0,0 +1,87 @@
+module TestExamples2DEulerMulticomponent
+
+using Test
+using Trixi
+
+include("test_trixi.jl")
+
+# pathof(Trixi) returns /path/to/Trixi/src/Trixi.jl, dirname gives the parent directory
+EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "tree_2d_dgsem")
+
+@testset "MHD Multi-ion" begin
+#! format: noindent
+
+@trixi_testset "elixir_mhdmultiion_ec.jl" begin
```

Comment:
Did you check how long the test runs? If it's more than 10s - can we restrict the time span for the tests?

Reply:
It runs for 1.44s on one core of Roci. See the [post above](https://github.com/trixi-framework/Trixi.jl/pull/2196#issuecomment-2531233021). 😉